dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/FilterJdbcPredicate.java

```diff
@@ -240,6 +248,50 @@ private static String rightOperandSql(QueryOperator operator, Parameter paramete
     return " :" + name;
   }
 
+  @Nonnull
+  private static String multiTextClause(
+      String column, QueryOperator operator, Parameter parameter) {
+    String unnestSql = "unnest(string_to_array(lower(" + column + "), ',')) AS val";
+    String trimmed = "trim(val)";
+    String param = parameter != null ? parameter.name() : null;
+
+    return switch (operator) {
+      case IEQ, EQ, IN ->
+          String.format("exists (select 1 from %s where %s in (:%s))", unnestSql, trimmed, param);
+      case NE, NEQ ->
+          String.format(
+              "not exists (select 1 from %s where %s in (:%s))", unnestSql, trimmed, param);
+      case LIKE, ILIKE ->
+          String.format("exists (select 1 from %s where %s like :%s)", unnestSql, trimmed, param);
```

Comment:
I am thinking why LIKE,ILIKE,NLIKE,NILIKE does not have the wildcards (%) concatenated before and after, (Similar to what you did for EW and SW). 

There is a test case down below that I added a comment on, if you modify the test case and if it fails, then probably it is because of this.